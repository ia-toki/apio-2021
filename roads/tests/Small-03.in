c10234cabdfd6eae47773fb6f163e0350fc797e1
191
162 59 1
36 41 1
95 139 1
6 60 1
177 41 1
21 93 1
132 93 1
8 26 1
109 26 1
104 139 1
153 59 1
75 53 1
23 53 1
17 59 1
81 72 1
46 26 1
3 60 1
42 139 1
146 59 1
175 139 1
166 72 20
1 41 1
49 60 1
80 26 1
154 139 1
182 93 1
89 26 1
127 59 1
125 139 1
26 72 20
73 53 1
56 72 1
113 60 1
130 59 1
124 41 1
91 60 1
108 60 1
145 184 1
38 26 1
86 72 1
131 139 1
67 72 1
189 53 1
159 41 1
39 166 1
114 139 1
11 93 1
176 41 1
44 53 1
161 184 1
141 60 1
71 26 1
167 166 1
58 53 1
156 184 1
101 72 1
105 53 1
85 60 1
76 72 1
186 184 1
187 59 1
119 41 1
87 60 1
168 166 1
31 93 1
184 72 20
97 26 1
188 139 1
116 53 1
142 59 1
178 41 1
63 139 1
50 53 1
33 166 1
158 139 1
64 93 1
82 59 1
183 184 1
93 72 20
171 59 1
120 166 1
47 139 1
151 60 1
37 184 1
149 72 1
143 139 1
40 139 1
29 139 1
7 53 1
126 59 1
62 59 1
13 53 1
174 166 1
144 59 1
19 26 1
122 26 1
169 184 1
45 184 1
185 72 1
5 93 1
28 41 1
10 53 1
111 60 1
83 93 1
70 93 1
27 59 1
129 26 1
115 72 1
59 72 20
53 72 20
98 59 1
30 166 1
16 93 1
74 139 1
139 72 20
9 53 1
112 41 1
157 60 1
84 60 1
2 166 1
99 26 1
66 93 1
25 60 1
103 93 1
72 100 1
60 72 19
79 53 1
96 41 1
68 59 1
179 41 1
181 41 1
163 93 1
140 59 1
92 139 1
78 166 1
34 184 1
51 166 1
102 166 1
52 26 1
48 166 1
155 26 1
118 26 1
43 184 1
35 166 1
77 166 1
22 60 1
128 53 1
117 59 1
152 139 1
41 72 20
14 26 1
164 166 1
24 41 1
165 93 1
173 26 1
57 93 1
65 41 1
88 166 1
134 93 1
4 72 1
54 166 1
160 184 1
107 184 1
172 60 1
110 41 1
138 41 1
15 184 1
147 53 1
148 60 1
20 184 1
32 26 1
133 59 1
0 26 1
121 184 1
90 166 1
170 93 1
94 139 1
137 53 1
135 93 1
69 53 1
12 166 1
18 53 1
180 184 1
136 184 1
150 41 1
106 60 1
61 41 1
55 93 1
123 184 1
190 184 1
