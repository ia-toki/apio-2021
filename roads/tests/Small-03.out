064bde9ff69ddc34b3b45c2d26d58873d85290d3
OK
360 332 304 276 248 220 192 164 136 109 99 89 79 69 59 49 39 29 19 9 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
